### ISSUE TITLE
Trailing space in CustomDateEditorRegistrar

### ISSUE THREAD

**Issue by marcelogore** [joergrech/KickstartWithBootstrap#36]
There is a blank space after the filename, just before the extension in CustomDateEditorRegistrar .groovy. I've checked the repository and it seems to have been there forever but my application won't run complaining that it cannot find the class. If I rename the file (wipe the blank space) it works perfectly. Is there a reason for the space being there?

**Comment by joergrech** [comment]
Thanks for the tip. That was really weird - I never had any problems with it and you are the first one mentioning it. I develop using STS and GGTS (eclipse) on Windows with oracle/sun' Java and ran the plugin and systems using it on windows7/8, MacBooks, CloudFoundry, Heroku, and Tomcat 6/7. Are you using special hardware, os, development environment or maybe a special java compiler (OpenJava, etc)?

**Comment by marcelogore** [comment]
Thank you for taking the time to look into it! There's nothing weird with my setup, as far as I know. I'm running Ubuntu 10.04 on two different x86 based notebooks, using SSTS 3.2.0 based on Eclipse Juno 4.2. Admittedly, the JDK is a bit old: 1.6.0_b22.
What's weirdest of all is that it did work when I started my project and I did build a whole new project from scratch and it worked, even with the space there. By the way, it's a Mavenized Grails 2.2.1 project.
Could it have something to do with the project building on Linux rather than Windows?
